app/views/generic_files/upload/_to_collection.html.erb

```diff
@@ -0,0 +1,6 @@
+<p>
+  <%= @user_collections = [] if @user_collections.nil? %>
```

Comment:
Line 2 was added as a fix for failing test [_browse_everything.html.erb_spec.rb](https://github.com/projecthydra/sufia/blob/master/spec/views/generic_file/_browse_everything.html.erb_spec.rb).  The controller always adds the default 'Select collection...', such that @user_collections is never nil.  But the test renders the page without going through the controller's #new and thus @user_collections is nil.  Maybe there is a better fix for the failing test.